tingbot/graphics.py

```diff
@@ -410,7 +395,7 @@ def _get_frames(self, pil_image):
             else:
                 palette = base_palette
 
-            pygame_image = pygame.image.fromstring(pil_image.tostring(), pil_image.size, pil_image.mode)
+            pygame_image = pygame.image.fromstring(pil_image.tobytes(), pil_image.size, pil_image.mode)
```

Comment:
What's the thinking behind this change? I'm getting `AttributeError: tobytes` on my machine.